learn/getting_started/quick_start.md

```diff
@@ -51,14 +51,14 @@ These commands launch the **latest stable release** of Meilisearch.
 
 ```bash
 # Fetch the latest version of Meilisearch image from DockerHub
-docker pull getmeili/meilisearch:v0.27.2
+docker pull getmeili/meilisearch:v0.28.0
```

Comment:
```suggestion
docker pull getmeili/meilisearch:v0.28
```

Reply:
Following my explanation here: https://github.com/meilisearch/documentation/pull/1742#issue-1273730059

Reply:
And I'm not sure this is useful to ask the users to fetch the version since you are asking to use a specific tag. Fetching is useful when you were using `latest` and you want the "new" `latest`

Reply:
Yes you're right, this command is not a real pain point for the docker users and can save some tricky use cases, let's keep it!